src/check_ast.rs

```diff
@@ -692,6 +695,9 @@ where
                 if match_name_or_attr(value, "Literal") {
                     self.in_literal = true;
                 }
+                if match_name_or_attr(value, "Annotated") {
+                    self.in_pep593_annotated = true;
+                }
```

Comment:
As a side note: it's totally valid to do `from typing import Literal as Foo`, I think this is just checking the name of the variable right?

Reply:
Yeah that's right. These checks are a bit limited in that they don't follow reassignment for these identifiers.